README.md

```diff
@@ -125,6 +126,7 @@ If you still need more speed, Arduino 1.0.5 produces slightly faster code.
 
 ## Caveats and Workarounds
 
+* AVR
```

Comment:
In the current version of the doc, this section is no longer AVR specific. (Yes, millis() and delay() are available on AVR, too, today, but it is not smart to use them - delay in particular - on any architecture)